conduct.md

```diff
@@ -15,7 +15,12 @@ testing fork
 * Participation
 * Conduct
 * Decisions
-* Postmortems
+* Postmortems:
+Meet in the first part of lab section to discuss where we are currently at in the given project.
+ One person can volunteer or someone can be picked to be our "leader" for the days     discussion.
+ Once ready members of the team can ask questions, bring up problems and errors that occurred.
+ After all the problems are sorted out we then talk about what worked well as a group and what we need to improve on for this upcoming week.
+ After finishing a project we should have a team monitor the program. In this they will check if any bugs or errors have risen and maintain code functionality.
```

Comment:
I believe we'll be using GitHub Issues to track bugs throughout the development process, so this might not fit into post-mortem

Reply:
I think it's still useful to reassess any issues that might have arisen during the "downfall" of whatever we're doing the postmortem on.